DOM_creation/Lesson3/main.js

```diff
@@ -0,0 +1,22 @@
+const ul = document.getElementById("js-ul");
+
+const a1 = document.createElement("a");
+a1.href = "a1.html";
+
+const a2 = document.createElement("a");
+a2.href = "a2.html";
+
+const img1 = document.createElement("img");
+img1.src = "/img/bookmark.png";
+
+const img2 = document.createElement("img");
+img2.src = "/img/message.png";
```

Comment:
IMO
Since there are similar operations, it is preferable to be able to loop the process from here.
What about making an array of parts that cannot be commonized?

JUST EXAMPLE
``
const imgSrc = ["./img/bookmark.png", "./img/message.png"];
``

And I feel I can loop it around with a for statement.

Reply:
Hi Chihiro-san,
Thank you for your feedback, I replaced forEach to for loop based on your advice. 
Please see my latest commitment